libsession/src/main/java/org/session/libsession/messaging/sending_receiving/ReceivedMessageHandler.kt

```diff
@@ -266,6 +266,9 @@ private fun handleNewClosedGroup(sender: String, sentTimestamp: Long, groupPubli
     val groupID = GroupUtil.doubleEncodeGroupID(groupPublicKey)
     if (storage.getGroup(groupID) != null) {
         // Update the group
+        // clearing zombie list if the group was not active before the update is received
+        if (!storage.isGroupActive(groupID))
+            storage.setZombieMembers(groupID, listOf<String>().map { Address.fromSerialized(it) })
```

Comment:
You can just do `listOf()` here :) . Also, `if` statement like this (without curly braces) can be misleading sometimes. Usually better to be explicit and add the curly braces rather than depend on indentation.